cli/module_build_test.go

```diff
@@ -150,24 +154,32 @@ func TestModuleGetPlatformsForModule(t *testing.T) {
 				},
 			}}, nil
 		},
-	}, &map[string]any{}, "token")
+	}, map[string]any{}, "token")
 	platforms, err := ac.getPlatformsForModuleBuild("xyz123")
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, platforms, test.ShouldResemble, []string{"linux/amd64", "linux/arm64"})
 }
 
+// testChdir is os.Chdir scoped to a test.
+// Necessary because Getwd() fails if run on a deleted path.
+func testChdir(t *testing.T, dest string) {
+	t.Helper()
+	orig, err := os.Getwd()
+	test.That(t, err, test.ShouldBeNil)
+	os.Chdir(dest)
+	t.Cleanup(func() { os.Chdir(orig) })
```

Comment:
I like to think you're using `t.Cleanup` after you learned about it in the last Golang reading group discussion, but I'm not sure if you were there 😆 .

Reply:
pytest reading group alas